api/envoy/extensions/clusters/dns/v3/dns_cluster.proto

```diff
@@ -86,5 +86,5 @@ message DnsCluster {
 
   // If true, perform :ref:`logical DNS discovery <arch_overview_service_discovery_types_logical_dns>`.
```

Comment:
Please update this comment to say something like this:

```
// If true, all returned addresses are considered to be associated with a single endpoint,
// which maps to :ref:`logical DNS discovery <arch_overview_service_discovery_types_logical_dns>`
// semantics.  Otherwise, each address is considered to be a separate endpoint, which maps to
// :ref:`strict DNS discovery <arch_overview_service_discovery_types_strict_dns>` semantics.
```